<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagePlugins</class>
 <widget class="QDialog" name="ManagePlugins">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage plugins</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="rbLoadAll">
     <property name="text">
      <string>Load all plugins on startup</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rbLoadSelected">
     <property name="text">
      <string>Select plugins to load:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabTools">
      <attribute name="title">
       <string>Tools</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QListWidget" name="listTools"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabFilters">
      <attribute name="title">
       <string>Filters</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QListWidget" name="listFilters"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
